solana-programs/anchor/audius-data/tests/audius-data.ts

```diff
@@ -1115,8 +1027,158 @@ describe("audius-data", function () {
     });
   });
 
+  it("delegate creates a track (manage entity) + all validation errors", async function () {
+    // create user and delegate
+    const delegate = await testCreateUserDelegate({
```

Comment:
nit but i think it would be clearer if we create the user and then create the delegation in two separate calls - OR rename this (breaking single responsibility principle) to `createUserAndDelegate`. My preference is for the former, where the createDelegate test helper takes in a user as arg.